searcher.py

```diff
@@ -74,8 +74,13 @@ def search(self, client, output_path_guest, topic_path_guest, test_split_path_gu
 
             # Time empty search
             search_args['topic']['path'] = os.path.join(topic_path_guest, single_query_file)
-            container = client.containers.run("{}:{}".format(self.config.repo, save_tag), command.format(json.dumps(json.dumps(search_args))), volumes=volumes,
+            if self.config.gpu:
+                container = client.containers.run("{}:{}".format(self.config.repo, save_tag), command.format(json.dumps(json.dumps(search_args))), volumes=volumes,
+                                              detach=True, runtime='nvidia')
```

Comment:
Can we change this (and the other instances) to have a variable called `runtime` that is either set to `nvidia` (if `--gpu` is passed) and `None` otherwise (if `--gpu` isn't passed)? Just removes some code duplication...